model-metadata/epiENGAGE-ensemble_mean.yml

```diff
@@ -20,7 +20,8 @@ designated_model: true
 team_funding: ""
 methods: "An equally weighted mean ensemble that takes the mean at each quantile level of all eligible forecasts."
 methods_url: "https://hubverse-org.github.io/hubEnsembles/articles/hubEnsembles.html"
-data_sources: "ILI NYC ED Visits, we exclude two seasons (20/21 and 21/22) and partial from the 19/20 season (removing data after March 2020), Flu TX ED visits pct from CDC"
+data_sources: "Using latest_data.csv from MetroCast Hub.  ILI NYC ED Visits, we exclude two seasons (20/21 and 21/22) and partial from the 19/20 season (removing data after March 2020)"
 ensemble_of_hub_models: true
 repo_url: "https://github.com/reichlab/flu-metrocast"
+local_fit_jointly: true
```

Comment:
I'm not quite sure what the answer should be here, but I think maybe it should be false. 
> A boolean (either TRUE or FALSE) where TRUE indicates that that the local jurisdictions (e.g. the HSAs within a state) were fit jointly and FALSE indicates that each unique location was fit independently.

given that we don't have control over whether the ensemble is fit jointly or not and that any model that is not fit jointly therefore would make it false, I suggest making this false.